bin/snapshot-backups/public-snapshot-create

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$0")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+
+export BUCKET_SOURCE_FOLDER=public
```

Comment:
I probably gave you a bad steer on this one. While we have likely-looking files at public/, the files we want are actually in cms/public/. 

I tried uploading an image via the dev CMS and it created a new path under cms/public/ but not under public/. 

This probably has implications for the restore script as well.

Reply:
@akf - oh dang... that's not good.  I'll modify the paths.